spec/acceptance/courses/enrollments_request_spec.rb

```diff
@@ -1,5 +1,5 @@
 require "rails_helper"
 
-RSpec.describe "Api::V1::Courses::Enrollments", type: :request do
+RSpec.describe "/courses/enrollments", type: :request do
```

Comment:
I think we should rename this file and all others from 
`spec/acceptance/courses/enrollments_request_spec.rb` to
`spec/acceptance/courses/enrollments_spec.rb`

What do you think about it?